testing-modules/mocks/src/test/java/com/baeldung/jimfs/FilePathReaderTest.java

```diff
@@ -0,0 +1,48 @@
+package com.baeldung.jimfs;
+
+import com.google.common.jimfs.Configuration;
+import com.google.common.jimfs.Jimfs;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.nio.file.FileSystem;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
```

Comment:
Hey @lukaszrys - This class causes the build to break I think:
`
[INFO] PMD Failure: com/baeldung/jimfs/FilePathReaderTest.java:14 Rule:UnitTestMustFollowNamingConventionRule Priority:3 Unit test class names need to end in UnitTest, integration tests with IntegrationTest, etc.
`

Reply:
@JonCook thanks!